tests/rest_api/rest_api.postman_collection.json

```diff
@@ -13451,60 +13451,6 @@
 					},
 					"response": []
 				},
-				{
-					"name": "Setparam password",
```

Comment:
I've removed this test cause a password cannot be changed during a defined delay